lifecycle-operator/controllers/lifecycle/keptntask/job_utils.go

```diff
@@ -138,3 +144,10 @@ func (r *KeptnTaskReconciler) generateJob(ctx context.Context, task *klcv1alpha3
 
 	return job, nil
 }
+
+func getServiceAccount(serviceAccount *klcv1alpha3.ServiceAccountSpec) error {
+	if serviceAccount == nil {
+		return errors.New("no serviceAccount is configured")
+	}
+	return nil
```

Comment:
this function always returns nil, that is not the desired behavior

Reply:
hey @odubajDT, can I use slog here to treat it as info